samza-core/src/main/java/org/apache/samza/container/grouper/task/GroupByContainerCount.java

```diff
@@ -211,7 +220,7 @@ private void validateTasks(Set<TaskModel> tasks) {
   private List<TaskGroup> createContainers(int startContainerId, int endContainerId) {
     List<TaskGroup> containers = new ArrayList<>(endContainerId - startContainerId);
     for (int i = startContainerId; i < endContainerId; i++) {
```

Comment:
I think there is more than one place in this Grouper where the integer assumption is used. The intention was to re-write this grouper with the new standalone. I will make sure there is a JIRA for this. Thanks :)